README.md

```diff
@@ -1 +1,54 @@
-# second-proj
\ No newline at end of file
+# second-proj
+in this project i created a hello microverse page using basic html and css. No complex code was done here.
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+## Live Demo (if available)
+
+## Getting Started
+To get a local copy
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [optional ] Good job for using the Readme file from the [Microverse Readme Tempelate](https://github.com/microverseinc/readme-template) as described in the [Project Requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements) and please try customize it according to your project. 🤗